gitlab/resource_gitlab_project_test.go

```diff
@@ -581,6 +581,36 @@ resource "gitlab_project" "foo" {
 	})
 }
 
+func TestAccGitlabProject_initializeWithCustomCIPath(t *testing.T) {
```

Comment:
Usually we try to only create new test functions if they require some unique setup. It looks like this could be added by adding the new attribute in the first two TestSteps in the existing TestAccGitlabProject_basic test. Other than that nitpick it looks good!

Reply:
Yay, my initial uncommitted changes were made to the first test function based on that simple assumption, however I ended up creating a new test case function for it as I have never contributed to the project before. I'll take a look tomorrow morning and correct as per your comments

Reply:
@armsnyder I've made the appropriate changes and ran the Acc. tests locally. Seems Ok

Reply:
Thanks, I appreciate it.